src/components/Chat/ChatInput/ChatInputMessage.tsx

```diff
@@ -138,14 +142,14 @@ export const ChatInputMessage = ({
       return;
     }
 
-    if (!content) {
+    if (isInputEmpty) {
       alert(t('Please enter a message'));
       return;
     }
```

Comment:
I think we don't want this alert and should just like this:
```javascript
   if (messageIsStreaming || isInputEmpty) {
        return;
   }
```
plus in `SendMessageButton` we need to add check for error cases to avoid sending on Enter click:
```
onClick={!isShowDisabled ? handleSend : undefined}
```

Reply:
Done slightly in different way